src/pty.c

```diff
@@ -48,18 +48,24 @@
 
 static struct termios stdin_attr;
 static struct winsize stdin_ws;
+static int stdin_is_tty = 1;
```

Comment:
It actually should have had the = 1 removed in the first part of the else.  The open_master_pty is not always called in the case when you attach separately.   I can invert the logic though to get rid of the else and provide a new patch.